{{cookiecutter.project_dirname}}/terraform/modules/kubernetes/deployment/main.tf

```diff
@@ -169,6 +169,10 @@ resource "kubernetes_deployment_v1" "main" {
               }
             }
           }
+          env {
+            name  = "CACHE_URL"
+            value = "$(REDIS_URL)?key_prefix=${var.environment_slug}"
```

Comment:
Instead of renaming the environmental variable to `REDIS_URL` I would leave the naming at `CACHE_URL` and add an environmental variable e.g. `CACHE_URL_PREFIX` and modify django's `settings.py` to combine CACHE_URL + CACHE_URL_PREFIX in Remote configurations

Reply:
There was no renaming. The intervention is intentionally as clean as possible. A `REDIS_URL` environment variable was added, containing the clean Redis server url for whatever package other than Django might need it. The `CACHE_URL` variable is left as-is. In my opinion, an intervention on the Django side would be more invasive and much less clean.